src/pages/Admin/Users/Delete/index.jsx

```diff
@@ -70,6 +71,19 @@ const DeleteUser = () => {
     }
   };
 
+  /* Defaults for the dropdown */
+  const setDefaultsForDropdown = (res) => {
+    const userList = res.map((user, idx) => ({
+      ...user,
+      disabled: !(idx !== 0 && idx !== 1),
```

Comment:
Can we have the `disabled` property based on username rather than index? Feels to me the index depends on REST API and it can change.

Reply:
Yes, my initial thought was to do the same, but I wasn't sure whether there is any possibility in future that we might change the default username then we have to do the code changes again. 

Keeping it index based would be an advantage, as I tested it multiple times and the default users were always at index 0 and 1 and it would never change since the users are sorted on the basis of ID, and we won't be deleting the default users causing no change in the position of default users in the users array. 

Any users which will be added manually will always have ID greater that IDs of each of default users, so the position will be always after index 1.

Reply:
The username as hard coded in FOSSology and they will not change. I can assure you that 😃 

But depending on index means depending on user id from FOSSology which again depends on database index. During migration, it can change (has happened before).

Reply:
Okay, then I'll do the required changes and add to the PR.

Reply:
I did the required changes, everything's working as expected. Also, I Amended the existing commit rather than creating a new one. @GMishx